examples/overlay-warnings/README.md

```diff
@@ -0,0 +1,11 @@
+# Overlay
+
+```shell
+node ../../bin/webpack-dev-server.js --open
+```
+
+## What should happen
+
+The script should open the browser and show a heading with "Example: overlay with warnings".
+
+In `app.js`, make a syntax error. The page should now refresh and show a full screen error overlay that shows the syntax error.
```

Comment:
Yeah, the problem is, how to trigger a warning.

I tried copying

```javascript
// This results in a warning:
// if(!window) require("./" + window + "parseable.js");
```

to the example from **modus-inline**, but that doesn't trigger a warning at all (not there either). Maybe something changed between webpack 1 and 2?